Documents/element_design.md

```diff
@@ -0,0 +1,46 @@
+Designing a UI element for a test automation framework like JDI-Light involves several steps to ensure the element is reusable, robust, and integrates well with the existing framework:
+
+### 1: Identify the Element
+- **Research:** Understand the UI component you want to design. For a material UI design, you might be looking at elements like buttons, cards, dialogs, sliders, etc. Use the JDI-Light, Bootstrap or Material Design guidelines as a reference.
+    - [JDI-Light](https://jdi-docs.github.io/jdi-light/?java#theory)
+    - [Bootstrap](https://getbootstrap.com/docs/5.3/components/) 
+    - [Material 2](https://m2.material.io/components?platform=web)
+    - [Material 3](https://m3.material.io/components?platform=web)
+
+- **Requirement Analysis:** Determine the functionalities and states this element can have. For instance, a button may have different states like hover, disabled, or pressed. For this step it is also good to use Material Design guidelines referencesd above.
```

Comment:
May be can add: image that you have this element in your project site, what should be testable for it?